robotbase/src/main/java/com/team2357/frc2022/commands/ClimberHookTogglePivotCommand.java

```diff
@@ -0,0 +1,31 @@
+package com.team2357.frc2022.commands;
+
+import com.team2357.frc2022.subsystems.ClimberSubsystem;
+import com.team2357.lib.commands.CommandLoggerBase;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+
+/**
+ * Command to toggle dynamic hooks open and close
+ * 
+ * @category Climber
+ */
+public class ClimberHookTogglePivotCommand extends CommandLoggerBase {
```

Comment:
Let's make a general purpose toggle command so we can use it with the set command.